tests/test_utils.py

```diff
@@ -395,3 +399,52 @@ def test_no_string(self):
 
         with self.assertRaisesRegex(TypeError, UNACCENT_TYPE_ERROR):
             unaccent_string(1234)
+
+
+class TestValidateField(TestCase):
+    """Unit tests for validate_field"""
+
+    def test_validate_string(self):
+        """Check valid and invalid fields"""
+
+        result = validate_field('test_field', 'Test')
```

Comment:
This function returns nothing. I'm not sure if we should compare it with `None`.

Reply:
Now this method is called twice with valid fields and using no assertions. If the values are valid, no exception has to be raised.